internal/checker/jsx_runtime_test.go

```diff
@@ -0,0 +1,33 @@
+// Test to ensure jsx-runtime imports are allowed in declaration files
```

Comment:
Do not create new unit tests, use compiler tests like in testdata/cases, or fix existing tests in the submodule. See your instructions on how to run tests.